ui/ui-frontend/projects/archive-search/src/app/archive/archive-preview/archive-unit-objects-details-tab/archive-unit-objects-details-tab.component.spec.ts

```diff
@@ -42,12 +50,29 @@ describe('ArchiveUnitObjectsDetailsTabComponent tests', () => {
   const clipboardSpy = createSpyObj<Clipboard>('Clipboard', ['copy']);
   const archiveServiceSpy = createSpyObj<ArchiveService>('ArchiveService', ['downloadObjectFromUnit', 'getObjectById']);
 
+  const tenantSelectionServiceSpy = jasmine.createSpyObj('TenantSelectionService', {
+    getSelectedTenant: {
+      name: 'tenantName',
+      identifier: 2,
+      ownerId: 'owner',
+      customerId: 'customer',
+      enabled: true,
+      proof: false,
+      readonly: true,
+      ingestContractHoldingIdentifier: 'string',
+      itemIngestContractIdentifier: 'string',
+      accessContractHoldingIdentifier: 'string',
+      accessContractLogbookIdentifier: 'string',
+    },
+  });
```

Comment:
Ce n'est pas suffisant ?
```suggestion
  const tenantSelectionServiceSpy = createSpyObj('TenantSelectionService', ['getSelectedTenant']);
```

Reply:
Il s'attend au champ identifier

Reply:
Ma remarque est qu'on a 2 fois l'objet: une fois dans le spy, une autre fois plus bas. Je voulais éviter la duplication.